features/steps/thens/relations.py

```diff
@@ -40,6 +40,24 @@ def step_impl(context, entity, condition, directness, other_entity):
 
     misc.handle_errors(context, errors)
 
+@then('Each {entity} {condition} be a part of {other_entities}')
+def step_impl(context, entity, condition, other_entities):
+    stmt_to_op = ['must', 'must not']
+    assert condition in stmt_to_op
+    expect_part = True if condition == 'must' else False
```

Comment:
`else False` makes it potentially dangerous for future work, e.g. if new `stmt_to_op` will be introduced and this conditional not amended accordingly. Proposal: default should throw a not implemented exception (or similar).